R/wb_styles.R

```diff
@@ -873,12 +877,12 @@ set_cell_style <- function(wb, sheet, cell, value) {
 create_dxfs_style <- function(
     font_name      = "Calibri",
     font_size      = "11",
-    font_color     = c(rgb = "FF9C0006"),
+    font_color     = wb_color(rgb = "FF9C0006"),
```

Comment:
Do we want to use a default font color in `options()` like we have for `options(openxlsx2.borderColor)`?

Reply:
I'm a bit undecided. 
1. I really really dislike how the function calls look, if they have all the `options(openxlsx2.xxx)` calls. Changing things under the hood, if not `NULL` seems like a bad idea too. 
2. Maybe we could go with a `name = "default"` arg. That will call the `options(openxlsx2.borderColor)` under the hood? But than again, we'd have to make sure that everywhere a "default" option is possible. That easily turns into a plethora of changes.